modules/tensor_mechanics/test/tests/umat/elastic.i

```diff
@@ -0,0 +1,111 @@
+# Testing the UMAT Interface - creep linear strain hardening model using the small strain formulation - visco-plastic material.
+# Note that this isn't a thermal or irradiation creep model.
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 3
+    xmin = -0.5
+    xmax = 0.5
+    ymin = -0.5
+    ymax = 0.5
+    zmin = -0.5
+    zmax = 0.5
+  []
+[]
+
+[Functions]
+  [top_pull]
+    type = ParsedFunction
+    value = t/100
+  []
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    add_variables = true
+    strain = FINITE
+  []
+[]
+
+[BCs]
+  [y_pull_function]
+    type = FunctionDirichletBC
+    variable = disp_y
+    boundary = top
+    function = top_pull
+  []
+  [x_bot]
+    type = DirichletBC
+    variable = disp_x
+    boundary = left
+    value = 0.0
+  []
+  [y_bot]
+    type = DirichletBC
+    variable = disp_y
+    boundary = bottom
+    value = 0.0
+  []
+  [z_bot]
+    type = DirichletBC
+    variable = disp_z
+    boundary = front
+    value = 0.0
+  []
+[]
+
+[Materials]
+  # active for umat calculation
```

Comment:
When I first read this, I had no idea what the comments on line 63 and 71 were about.

I think it would be helpful to say a little more -- that the same file is used to compare the MOOSE and UMAT models, activating specific ones with cli args.